raiden/api/v1/encoding.py

```diff
@@ -62,6 +66,28 @@ def to_url(self, value):
         return to_checksum_address(value)
 
 
+class HexChannelIdConverter(BaseConverter):
+    def to_python(selfself, value):
+        if value[:2] != '0x':
+            raise ValidationError()
```

Comment:
Shouldn't we provide an error message here and below?

Reply:
I thought about that, but it wasn't done in the old code. I'll add it